lib/model/model.go

```diff
@@ -981,6 +981,20 @@ func (m *Model) GetIgnores(folder string) ([]string, []string, error) {
 	return lines, patterns, nil
 }
 
+func (m *Model) GetFolderFiles(folderID string) *db.FileSet {
```

Comment:
It's an exported method. Those can't be dead code eliminated so it makes sense that metalinter doesn't see this, when thinking about it. :)